riscv/v_ext_macros.h

```diff
@@ -1948,17 +1948,17 @@ reg_t index[P.VU.vlmax]; \
   switch (P.VU.vsew) { \
     case e16: \
       { VI_VFP_CVT_LOOP(CVT_INT_TO_FP_PARAMS(16, 16, sign), \
-        { p->extension_enabled(EXT_ZVFH); }, \
+        { require(p->extension_enabled(EXT_ZVFH)); },   \
```

Comment:
I think this was a bug in the old code, the macro extension wouldn't wrap the `p->extension_enabled` with a `require`, from what I could tell.

Reply:
Yeah, looks that way.